namespaces/init.go

```diff
@@ -48,6 +48,15 @@ func Init(container *libcontainer.Config, uncleanRootfs, consolePath string, pip
 		pipe.Close()
 	}()
 
+	if container.Namespaces.Contains(libcontainer.NEWUSER) {
+		err = initUserNs(container, uncleanRootfs, consolePath, pipe, args)
```

Comment:
If we anyway return `err`, we can just return for both branches of condition and omit `else`